flink-optimizer/src/test/java/org/apache/flink/optimizer/operators/CoGroupGlobalPropertiesCompatibilityTest.java

```diff
@@ -35,7 +37,7 @@ public void checkCompatiblePartitionings() {
 			final FieldList keysLeft = new FieldList(1, 4);
 			final FieldList keysRight = new FieldList(3, 1);
 			
-			CoGroupDescriptor descr = new CoGroupDescriptor(keysLeft, keysRight);
+			CoGroupDescriptor descr1 = new CoGroupDescriptor(keysLeft, keysRight);
```

Comment:
Can't we use this object for the new test as well? Then we would not need to rename it.